crates/router/src/connector/stax/transformers.rs

```diff
@@ -282,6 +285,7 @@ pub struct StaxPaymentsResponse {
     child_captures: Vec<StaxChildCapture>,
     #[serde(rename = "type")]
     payment_response_type: StaxPaymentResponseTypes,
+    reference: Option<String>,
```

Comment:
Same here 
```suggestion
    idempotency_id: Option<String>,
```

Reply:
Thank You for the feedback @srujanchikke, I have made the necessary changes.